index.bs

```diff
@@ -1390,7 +1390,7 @@ When the [=view/active=] flag of any [=view=] in the [=XRSession/list of views=]
 Primary and Secondary Views {#primary-and-secondary-views}
 ----------
 
-A [=view=] is a <dfn>primary view</dfn> when rendering to it is necessary for an immersive experience.
+A [=view=] is a <dfn>primary view</dfn> when rendering to it is necessary for an immersive experience. [=Primary views=] MUST be [=view/active=] for the entire duration of the {{XRSession}}.
 
 A [=view=] is a <dfn>secondary view</dfn> when it is possible for content to choose to not render to it and still produce a working immersive experience. When content chooses to not render to these views, the user-agent MAY be able to reconstruct them via reprojection.
```

Comment:
The other half of the equation would be the following requirement:

```suggestion
A [=view=] is a <dfn>secondary view</dfn> when it is possible for content to choose to not render to it and still produce a working immersive experience. When content chooses to not render to these views, the user-agent MAY be able to reconstruct them via reprojection. Secondary views MUST NOT be [=view/active=] unless the "[=secondary view/secondary-views=]" feature is enabled.
```

Reply:
That's unnecessary, secondary views don't _exist_ if that feature is not enabled.

Reply:
Oh, actually we never say that. I'll add this.